eBookReader/chat.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Q&A</title>
+	<link rel="stylesheet" type="text/css" href="styleIndex.css"/>
+	<link rel="stylesheet" type="text/css" href="debugStyle.css"/>
+</head>
+<body>
+<header>
+	<p class = "name">John Doe</p>
+	<p class = "userStats">
+		Contribution points: 91
+		<a href="#" class="iconButton">?</a>
+	</p>
+</header>
+
+<h2>According to the author, why do mass extinctions happen?</h2>
+
+<div class = "chatPanel">
+	<div class = "chatMessage">I believe that on page 3, the author clearly states that extinctions happen for three reasons.</div>
+	<div class = "contributePanel">
+		<img class = "centerBadge answerBadge" src = "orange triangle.png" alt = "Caution Triangle" height = "50" width = "50">
+		<a class = "button contributeButton" href="#" >Contribute</a>
+	</div>
+</div>
+
+<p class = "timeStamp chat">Anthony Stark | 2 hours ago</p>
+
+<div class = "chatPanel">
+	<div class = "chatMessage">I thought there were only 2.</div>
+	<div class = "contributePanel">
+		<img class = "answerBadge" src = "green check mark.png" alt = "Green Check Mark" height = "50" width = "50">
+		<img class = "answerBadge" src = "three orange dots.png" alt = "Three Dots" height = "50" width = "50">
+		<a class = "button contributeButton" href="#" >Contribute</a>
+	</div>
+</div>
+
+<p class = "timeStamp chat">Steve Rodgers | 15 minutes ago</p>
+
+<div class = "chatPanel">
+	<div class = "chatMessage">Because the author is being sarcastic.</div>
+	<div class = "contributePanel">
+		<a class = "noBadge button contributeButton" href="#" >Contribute</a>
+	</div>
+</div>
+
+<p class = "timeStamp chat">Steve Rodgers | 14 minutes ago</p>
+
+<div class = "chatPanel">
+	<div class = "noContribute">
+		<img class = "centerBadge answerBadge" src = "green check mark.png" alt = "Green Check Mark" height = "50" width = "50">
+	</div>
+	<div class = "userMessage">Steve is right. The third reason, lack of flossing, was just a joke by the author.</div>
+</div>
+
+<p class = "timeStamp user">Me | Now</p>
+
+<div class = "replyPanel">
```

Comment:
I forgot to mention that we want the replyPanel to be visible at all times, no matter where the user is at (right now, it shows only if you scroll to the bottom of the page). This is a very simple fix: you have to set the replyPanel's display property to fixed (it's currently relative). Then, to make sure that it doesn't cut the bottom of the page, give the last chatPanel (remember the :last-child pseudo rule?) a margin-bottom of probably around 60px.